_episodes/065-ed-page.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "CLAP"
+date: 2022-04-22T15:30:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e065-ed-page.mp3
+duration: "1:10:32"
+length: "67724852"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Ed Page](https://github.com/epage), maintainer of [CLAP](https://github.com/clap-rs/clap). Command Line Argument Parser (CLAP) is a library to help create CLI apps using Rust.
```

Comment:
I don't think Clap is generally captialized like that? I would go with the same style they use in their documentation.

Reply:
We should also fix that up in the timestamp section below.